src/templates/zombienet/polkadot-parachain.toml

```diff
@@ -0,0 +1,7 @@
+[[parachains]]
+id = 100
+
+    [parachains.collator]
```

Comment:
is this a convention for zombienet configuration to add indents?

Reply:
yes it seems to be a convention according to the [documentation](https://paritytech.github.io/zombienet/cli/spawn.html) and official [examples](https://github.com/paritytech/zombienet/blob/main/examples/0000-test-config-small-network.toml)